tests/utils/object_store/test_gs_object_store.py

```diff
@@ -63,11 +63,13 @@ def test_gs_object_store_integration_json_auth(expected_use_gcs_sdk_val=True, cl
 
 @pytest.mark.gpu
 @pytest.mark.remote
-def test_gs_object_store_integration_hmac_auth():
+def test_gs_object_store_integration_json_auth():
     with mock.patch.dict(os.environ):
-        if 'GOOGLE_APPLICATION_CREDENTIALS' in os.environ:
-            del os.environ['GOOGLE_APPLICATION_CREDENTIALS']
-        test_gs_object_store_integration_json_auth(expected_use_gcs_sdk_val=False, client_should_be_none=True)
+        if 'GCS_KEY' in os.environ:
+            del os.environ['GCS_KEY']
+        if 'GCS_SECRET' in os.environ:
+            del os.environ['GCS_SECRET']
+        test_gs_object_store_integration_hmac_auth(expected_use_gcs_sdk_val=True, client_should_be_none=False)
```

Comment:
so iiuc there was some change in gcs sdk that changes the expected values here?

Reply:
We reordered which method is checked first